src/components/Loader/PersonItem.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  mother: Person | string | null,
+  father: Person | string | null,
+  person: Person,
+  selectedPerson: string
+};
+
+export const PersonItem: React.FC<Props> = ({
+  mother,
+  father,
+  person,
+  selectedPerson,
+}) => {
+  const { sex, born, died } = person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': person.slug === selectedPerson,
+      })}
+    >
+      <td>
+        <PersonLink person={person} />
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        <>
+          {(mother && typeof mother === 'object')
+            && <PersonLink person={mother} />}
+          {(mother && typeof mother === 'string') && `${mother}`}
+          {!mother && '-'}
```

Comment:
Can't you move this logic to findRelative function and just return a proper value from it to use it here?